IdentityCore/src/util/NSDictionary+MSIDExtensions.h

```diff
@@ -25,11 +25,19 @@
 
 @interface NSDictionary (MSIDExtensions)
 
-+ (NSDictionary *)msidDictionaryFromQueryString:(NSString *)string;
+typedef NS_ENUM(NSInteger, MSIDEncodingType){
```

Comment:
where's this being used?

Reply:
Was going to be used but decided not to, as it adds needless more complexity